tests/manage/z_cluster/test_ceph_default_values_check.py

```diff
@@ -109,3 +116,31 @@ def test_validate_ceph_config_values_in_rook_config_override(self):
             f"The expected values are:\n{stored_values}\n"
             f"The cluster's Ceph values are:{config_data}"
         )
+
+    @post_ocs_upgrade
+    @skipif_ocs_version("<4.7")
+    @bugzilla("1951348")
+    @bugzilla("1944148")
+    @pytest.mark.polarion_id("OCS-2554")
+    def test_check_mds_cache_memory_limit(self):
+        """
+        Testcase to check mds cache memory limit post ocs upgrade
+
+        """
+        mds_a_dict, mds_b_dict = get_mds_config_value()
+        if (mds_a_dict["value"] and mds_b_dict["value"]) == "4294967296":
```

Comment:
It will be clearer if you use a var like for example: `expected_mds_value="4294967296"`.

Reply:
done, change accordingly